libsrc/model_data_order.inc

```diff
@@ -0,0 +1,8 @@
+!STARTOFREGISTRYGENERATEDINCLUDE 'inc/model_data_order.inc'
+!
+! WARNING This file is generated automatically by use_registry
+! using the data base in the file named Registry.
+! Do not edit.  Your changes to this file will be lost.
```

Comment:
What is this about? What is use_registry?

Reply:
I'm not sure. I deleted this file and hardcoded the value instead of including it, since it's already hardcoded into the repo and there's no `Registry` file.